README.md

```diff
@@ -94,7 +94,7 @@ To use the graph api id method to fetch the share count you need to set up an ap
  * @var array
  */
 private static $configuration = [
-    'cacheClass' => 'Heise\\Shariff\\ZendCache',
+    'cacheClass' => 'Heise\Shariff\LaminasCache',
```

Comment:
I propose changing this to \Heise\Shariff\LaminasCache::class instead of fiddling with strings

Reply:
No problem. Updated